deepmd/nvnmd/entrypoints/mapt.py

```diff
@@ -0,0 +1,334 @@
+
+import numpy as np
+import logging
+
+from deepmd.env import tf
+from deepmd.utils.sess import run_sess
+
+from deepmd.nvnmd.utils.fio import FioDic
+from deepmd.nvnmd.utils.config import nvnmd_cfg
+from deepmd.nvnmd.utils.weight import get_normalize, get_rng_s, get_filter_weight
+from deepmd.nvnmd.utils.network import get_sess
+
+from deepmd.nvnmd.data.data import jdata_deepmd_input
+
+from typing import List, Optional
+
+log = logging.getLogger(__name__)
+
+
+class MapTable:
+    r""" Generate the mapping table describing the relastionship of
+    atomic distance, cutoff function, and embedding matrix.
+
+    three mapping table will be built:
+    .. math::
```

Comment:
![Screenshot from 2022-06-17 18-01-28](https://user-images.githubusercontent.com/9496702/174406530-7ba9b1ff-cc19-45ad-bb20-82d64098370b.png)

I think a blank line should be inserted before `.. math::`